icd/api/strings/generate_strings.py

```diff
@@ -67,6 +69,10 @@ def GetOpt():
                   type="string",
                   dest="workdir",
                   help="the work directory")
+    parser.add_option("-o", "--output_dir", action="store",
```

Comment:
could you change here too? '-o' -> '-d' or any other letter

Reply:
Done